package.json

```diff
@@ -36,6 +36,7 @@
   "homepage": "http://purecss.io",
   "main": "index.js",
   "browser": "build/pure-min.css",
+  "style": "build/pure.css",
```

Comment:
Just to confirm, do you want `pure.css` or the minified version?

Reply:
`pure.css` is correct. You would likely have a minifier in your postcss chain for production. This seems to be what most other css frameworks use as well.

* [Milligram](https://github.com/milligram/milligram/blob/master/package.json#L9)
* [Bulma](https://github.com/jgthms/bulma/blob/master/package.json#L8)
* [Bootstrap](https://github.com/twbs/bootstrap/blob/v4-dev/package.json#L66)

Reply:
Ok cool